commit ab344ac94677dc6297e54e579bb00b1fd427d83b
Author: amnesia <amnesia@boum.org>
Date:   Thu Sep 13 11:27:20 2012 +0200

    Raise the socket library timeout to 120 seconds.
    
    During testing, I have experienced many timeouts with the original (60 seconds)
    value, that don't happen with 120.

diff --git a/whisperBack/mail.py b/whisperBack/mail.py
index 73a89f4..856a6ef 100644
--- a/usr/share/pyshared/whisperBack/mail.py
+++ b/usr/share/pyshared/whisperBack/mail.py
@@ -68,7 +68,7 @@ def send_message_tls (from_address, to_address, message, host="localhost",
     # We set a long timeout because Tor is slow
     # XXX: this will not be necessary anymore under python 2.6, because it
     #      includes a timeout argument on smtplib
-    socket.setdefaulttimeout(60)
+    socket.setdefaulttimeout(120)
 
     # Send the message via our own SMTP server, but don't include the
     # envelope header.
